adafruit_lis3dh.py

```diff
@@ -387,7 +409,14 @@ class LIS3DH_I2C(LIS3DH):
 
     """
 
-    def __init__(self, i2c, *, address=0x18, int1=None, int2=None):
+    def __init__(
+        self,
+        i2c: I2C,
+        *,
+        address: Optional[int] = 0x18,
```

Comment:
Same thing regarding `Optional` here - it should just be `int` since it cannot be `None`.

Reply:
Tks see https://github.com/adafruit/Adafruit_CircuitPython_LIS3DH/pull/76/commits/59c669238f330e5703bb89641199c02303db38c7